### ISSUE TITLE
How do we organize training resources? 

### ISSUE THREAD

**Issue by cansavvy** [fhdsl/fhdsl.github.io#15]
We need to have designations for affiliations but we also want things to be user friendly to find.

**Comment by cansavvy** [comment]
Things we need to figure out:

1) How to denote which courses are "A new Hutch thing" vs "Hutch DaSL affiliated/sponsored/powered"
2) But more importantly, we need to organize courses in such a way that users can find resources and projects that are helpful to them. So organizing by topic or audience of the project's material.

**Comment by cansavvy** [comment]
Might make sense to make a table of sorts. 

I think on a SciWiki page these resources would also be listed but perhaps on that page it would be a fancy table that allows folks to search by topic or some other type of relevant categorization.

**Comment by vortexing** [comment]
Yeah, the search feature in SciWiki would be great to leverage.  But ultimately hutchdatascience.org should be the ground truth (however that works).  ;)
